### DISCUSSION TITLE
esp32-4827S043R

### DISCUSSION THREAD

**Discussion by fanatykCZ516** [suchmememanyskill/CYD-Klipper#90] [Ideas]
Hi bro, could you please support the esp32-4827S043R?

**Comment by suchmememanyskill** [comment]
@fanatykCZ516 

Try it out:
https://github.com/suchmememanyskill/CYD-Klipper/blob/dev/Manual_Install.md
https://github.com/suchmememanyskill/CYD-Klipper/actions/runs/8693617552
